docs/extras.md

```diff
@@ -5,23 +5,24 @@ title: Extras
-| `materialize`         | Add nav, responsive and compact helpers for the Materialize CSS [pagination component](https://materializecss.com/pagination.html)                     | [materialize.rb](https://github.com/ddnexus/pagy/blob/master/lib/pagy/extras/materialize.rb), [documentation](extras/materialize.md)                         |
-| `overflow`            | Allow for easy handling of overflowing pages                                                                                                           | [overflow.rb](https://github.com/ddnexus/pagy/blob/master/lib/pagy/extras/overflow.rb), [documentation](extras/overflow.md)                                  |
-| `plain`               | Add responsive and compact plain/unstyled helpers                                                                                                      | [plain.rb](https://github.com/ddnexus/pagy/blob/master/lib/pagy/extras/plain.rb), [documentation](extras/plain.md)                                           |
-| `searchkick`          | Paginate arrays efficiently avoiding expensive array-wrapping and without overriding                                                                   | [searchkick.rb](https://github.com/ddnexus/pagy/blob/master/lib/pagy/extras/searchkick.rb), [documentation](extras/searchkick.md)                            |
-| `semantic`            | Add nav, responsive and compact helpers for the Semantic UI CSS [pagination component](https://semantic-ui.com/collections/menu.html)                  | [semantic.rb](https://github.com/ddnexus/pagy/blob/master/lib/pagy/extras/semantic.rb), [documentation](extras/semantic.md)                                  |
-| `support`             | Extra support for features like: incremental, infinite, auto-scroll pagination                                                                         | [support.rb](https://github.com/ddnexus/pagy/blob/master/lib/pagy/extras/support.rb), [documentation](extras/support.md)                                     |
-| `trim`                | Remove the `page=1` param from links                                                                                                                   | [trim.rb](https://github.com/ddnexus/pagy/blob/master/lib/pagy/extras/trim.rb), [documentation](extras/trim.md)                                              |
+| Extra                 | Description                                                                                                                               | Links                                                                                                                                                        |
```

Comment:
could avoid the giant diff / make blame simler by not aligning the `|`

Reply:
true, but a non-aligned source makes me sick :)